src/components/chips/js/chipsController.js

```diff
@@ -716,6 +744,43 @@ MdChipsCtrl.prototype.onInputBlur = function () {
   }
 };
 
+/**
+ * Configure event bindings on input element.
+ * @param inputElement
+ */
+MdChipsCtrl.prototype.configureInput = function configureInput(inputElement) {
+  // Find the NgModelCtrl for the input element
+  var ngModelCtrl = inputElement.controller('ngModel');
+  var ctrl = this;
+
+  if (ngModelCtrl) {
+
+    // sync touched-state from inner input to chips-element
+    this.deRegister.push(
+      this.$scope.$watch(
+        function() {
+          return ngModelCtrl.$touched;
```

Comment:
You can combine this single watch into two. e.g.

```js
this.$scope.$watch(() => {
  return { touched: inputNgModelCtrl.$touched, dirty: inputNgModelCtrl.$dirty };
}, XXXX);
```

Reply:
But dont I need $watchCollection for this? And then separate afterwards anyway?

Reply:
Don't think a `$watchCollection` will work here, because it requires us to watch properties on the `$scope`. Using `$watch` and just having a different watch condition should work.

https://stackoverflow.com/questions/17550947/how-to-watch-multiple-variables

Reply:
If I do as suggested I run in an endless digest-cycle with karma. What speaks against having to simple watcher?

Reply:
Let's leave it as is, thanks for giving it a try.